tools/webpack/blocks.js

```diff
@@ -71,13 +85,15 @@ module.exports = function( env = { environment: 'production', watch: false, buil
 		'code',
 		'column',
 		'columns',
+		'comments-query-loop',
 		'cover',
 		'embed',
 		'freeform',
 		'group',
 		'heading',
 		'html',
 		'list',
+		'list-item',
```

Comment:
We miss one more block:

Post Author Biography

https://github.com/WordPress/gutenberg/pull/38269

Reply:
List Item is not used at the moment but we probably need to include it.

Reply:
Good spot! Just added in https://github.com/WordPress/wordpress-develop/pull/2564/commits/5e0bbf43be14a64c82332be4cc028c6237a3b84a